<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FSobelDialog</class>
 <widget class="QDialog" name="FSobelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sobel Kernel</string>
  </property>
  <widget class="QLabel" name="lblDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboDirection">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="cmdOk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblKernel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel Size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnKernel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkScharr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Scharr</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
